src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -58,71 +51,74 @@ const logger = createLogger("Order/Routes/Review/index.tsx")
 
 const OrdersReviewOwnerType = OwnerType.ordersReview
 
-@track()
-export class ReviewRoute extends Component<ReviewProps> {
-  @track<ReviewProps>(props => ({
-    action_type:
-      props.order.mode === "BUY"
-        ? Schema.ActionType.SubmittedOrder
-        : Schema.ActionType.SubmittedOffer,
-    order_id: props.order.internalID,
-    products: [
-      {
-        product_id:
-          props.order.lineItems?.edges?.[0]?.node?.artwork?.internalID,
-        quantity: 1,
-        price: props.order.itemsTotal,
-      },
-    ],
-  }))
-  async onSubmit(setupIntentId?: any) {
+export const ReviewRoute: FC<ReviewProps> = props => {
+  const { trackEvent } = useTracking()
+
+  const onSubmit = async (setupIntentId?: any) => {
+    const onSubmitEvent = {
```

Comment:
**suggestion/chore**: we might consider dropping 'on' and have this const named `submitEvent`.